src/components/mortgage-investment-compare/validated-number-input.js

```diff
@@ -0,0 +1,36 @@
+import _ from 'lodash'
+import React, { useState } from 'react'
+import { Input } from 'reactstrap'
+
+export default function (props) {
+  const [value, setValue] = useState(props.value)
+  const [isValid, setIsValid] = useState(true)
+
+  function onInputChange (event) {
+    const value = event.target.value
+    setValue(value)
+    const parsed = parseFloat(value)
+    if (!value || !_.isInteger(parsed)) {
+      setIsValid(false)
+      return
+    }
+    if (_.isNumber(props.min) && parsed <= props.min) {
+      setIsValid(false)
+      return
+    }
+    setIsValid(true)
+    props.onChange(props.name, parsed)
+  }
+
+  return (
+    <Input
+      invalid={!isValid}
+      min={props.min}
+      name={props.name}
```

Comment:
I noticed you were passing `max`. Would you want to add it here as well?

Reply:
👍 As it turns out though, it doesn't do anything. Maybe I should actually remove `min`

Reply:
@hdavidzhu it does. I have tested it. I'm not sure if you are understanding it's use correctly.

Reply:
will just add back in my next PR

Reply:
On the `reactstrap` Input? It works for a standard input, but when I tried using min, I don't think it prevented me from hitting the down button below min 🤷‍♂ but if it works for you

Reply:
maybe some browser issue @hdavidzhu ? let's test it next time we chat for the sake of curiosity. Works perfectly on my end